packages/media/src/test/calculateSrcSet.js

```diff
@@ -117,6 +117,74 @@ describe('calculateSrcSet', () => {
     expect(srcSet).toBe('large-url 400w');
   });
 
+  it('should sizes with spaces', () => {
+    const resource = {
+      src: 'default-url',
+      width: 500,
+      height: 500,
+      sizes: {
+        img1: { width: 100, height: 100, source_url: 'small url' },
+        img2: { width: 200, height: 200, source_url: 'medium url' },
+        img3: { width: 300, height: 300, source_url: 'large url' },
+      },
+    };
+
+    const srcSet = calculateSrcSet(resource);
+    expect(srcSet).toBe('large%20url 300w,medium%20url 200w,small%20url 100w');
+  });
+
+  it('should sizes with encoded spaces', () => {
```

Comment:
```suggestion
  it('should encode already encoded URLs', () => {
```

Although this test basically just tests `encodeURI` itself, so seems a bit overkill